language/english/blocks.php

```diff
@@ -3,7 +3,7 @@
 /**
  * Language strings for blocks (en)
  *
- * @package chess
+ * @package    chess
  * @subpackage language
  */
 
```

Comment:
Add opening phpDoc @ignore for closing statement at end of file
```suggestion
/**#@+
 * @ignore
 */
```